docs/mkdocs/docs/features/trailing_commas.md

```diff
@@ -0,0 +1,37 @@
+# Trailing Commas
+
+Like comments, this library does not support trailing commas in arrays and objects *by default*.
+
+You can pass parameter `ignore_trailing_commas` to `#!cpp true` in the parse function to allow trailing commas in arrays and objects. Trailing commas are ignored during parsing. Note that a single comma as the only content of the array or object (`[,]` or `{,}`) is not allowed.
+
+This library does not add trailing commas when serializing JSON data.
+
+!!! example
+
+    Consider the following JSON with trailing commas.
+
+    ```json
+    {
+        "planets": [
+            "Mercury",
+            "Venus",
+            "Earth",
+            "Mars",
+            "Jupiter",
+            "Uranus",
+            "Neptune",
+        ]
+    }
+    ```
+    
+    When calling `parse` without additional argument, a parse error exception is thrown. If `ignore_trailing_commas` is set to `#! true`, the trailing commas are ignored during parsing:
+
+    ```cpp
```

Comment:
Please also add this example to the documentation of the `parse` function in `parse.md`.